packages/web-components/fast-element/src/styles.ts

```diff
@@ -101,7 +101,7 @@ export class AdoptedStyleSheetsStyles extends ElementStyles {
     public removeStylesFrom(target: StyleTarget): void {
         const sourceSheets = this.styleSheets;
         target.adoptedStyleSheets = target.adoptedStyleSheets!.filter(
-            (x: CSSStyleSheet) => !sourceSheets.includes(x)
```

Comment:
Out of curiosity - what is the impetus for this change? Perf?

Reply:
When I was setting things up I was getting type errors on the old version. So, I changed this to avoid those errors. I think it was just wrong configuration at the time though, so it could be changed back. I just left it in. I'm not sure the perf implications...probably negligible for this scenario would be my guess.